ynput/ayon-houdini
# asset HDA LOP: Implement and use  select product dialog

## Changelog description
Use dialog to pick product and remove redundant Houdini parameter.

![Animation_88](https://github.com/ynput/ayon-houdini/assets/20871534/828d1764-cf49-4754-b0d3-dcc337a72f77)


## Additional info
There's a bug where the representation name is not updating. I can reproduce it in develop branch.
oh, there's an issue for it already https://github.com/ynput/ayon-houdini/issues/32


## Testing Notes:
1. use asset lop.
2. try the product picker.
3. everything should work as usual.
